src/lib/Behat/PageElement/ContentUpdateForm.php

```diff
@@ -6,6 +6,7 @@
  */
 namespace EzSystems\EzPlatformAdminUi\Behat\PageElement;
 
+use eZ\Publish\API\Repository\Values\Content\Content;
```

Comment:
Is this import used or can it be removed?

Reply:
No it's not used. I don't create any content using API. Thanks